tests/unit/main.c

```diff
@@ -35,6 +35,9 @@ static const char *testcase_name = NULL;
 
 typedef int (*ctest_filter_func)(struct ctest *);
 
+// If 0, then CTEST_LOG_INFO() and CTEST_LOG() will no-op.  Defaults to 0.
+int Ctest_verbosity = 0;
```

Comment:
Any reason  why this is declared as `int` rather than as `bool` ? (I think I dimly recall some issue with using `bool` as a type in ctests' `main.c` ... Maybe that's why this is an `int` ?)

You may want to doc that this is really being treated as a boolean variable.

It would be good to be consistent elsewhere where this variable is checked. In some places code does `if (Ctest_verbosity)` and in others it does `if (Ctest_verbosity > 0)`.

Reply:
Good point.  Changed to `_Bool` on latest.